forum/views.py

```diff
@@ -72,14 +73,19 @@ def create_thread(request):
 def create_response(request):
 	form = ResponseForm(request.POST)
 	if(form.is_valid()):
+		thread = Thread.objects.get(id=request.POST.get('thread'))
```

Comment:
Is it better to use `request.POST` or `form` here?

Reply:
That's directly copied from your old code (it's just taking an inline definition and splitting it into a variable). In short, no.
![2018-09-17-211018_771x341_scrot](https://user-images.githubusercontent.com/4217801/45647628-3d4fe980-babe-11e8-92e5-5db965a365f1.png)

Reply:
https://docs.djangoproject.com/en/2.1/ref/forms/api/#django.forms.Form.cleaned_data perhaps?